docs/src/pages/changelogs/[...slug].astro

```diff
@@ -0,0 +1,35 @@
+---
+import Base from '../../layouts/Base.astro';
+import { getCollection } from 'astro:content';
+
+export const getStaticPaths = async () => {
+  return (await getCollection('changelogs')).map((entry) => ({
+    params: { slug: entry.data.version.replace(/\./g, '-') },
+    props: { entry }
+  }));
+};
+
+const { entry } = Astro.props;
+const { Content } = await entry.render();
+---
+
+<Base
+  title={`Version ${entry.data.version} - Eludris Changelogs`}
+  description={`The changes that occured in Eludris for version ${entry.data.version}`}
+  slug={entry.data.version}
+>
+  <h1>Version {entry.data.version} Changes</h1>
+  <Content />
+  <div style="display: flex; justify-content: center;">
+    <table style="width: auto;">
+      <thead>
+        <th style="border-bottom-left-radius: 5px; border-bottom-width: 2px;">Update PR</th>
+        <th style="border-bottom-right-radius: 5px; border-bottom-width: 2px;"
+          ><a href={`https://github.com/eludris/eludris/pull/${entry.data.pr}`}
+            >eludris#{entry.data.pr}</a
+          ></th
+        >
```

Comment:
Genuine question-- what the heck is going on with the tag closing here?

Reply:
Prettier